registration/utils.py

```diff
@@ -35,4 +37,77 @@ def verify_user_emails(user: User, request: HttpRequest, *email_addresses: str):
     for email_address in email_addresses:
         if not EmailAddress.objects.filter(email=email_address).exists():
             email = EmailAddress.objects.create_unconfirmed(email_address, user)
-            send_email_verification(email, request)
+            send_email_verification(email)
+
+
+def send_welcome_email(application: BetaTestApplication) -> None:
+    """
+    Send a welcome email to a new user after they have confirmed their email
+    addresses.
+    """
+    user = application.user
+    context = dict(
+        user_name=user.profile.full_name,
+        lms_link=application.instance.get_domain('lms'),
+        studio_link=application.instance.get_domain('studio'),
+        # TODO: add in once implemented
+        customise_link='',
+    )
+    html_email_helper(
+        template_base_name='emails/welcome_email',
+        context=context,
+        subject=settings.WELCOME_EMAIL_SUBJECT,
+        recipient_list=(user.email,)
+    )
+
+
+def send_welcome_email_test(email: str) -> None:
+    """
+    Send a test welcome email.
+    """
+    context = dict(
+        user_name="John Doe",
+        lms_link="http://my.edx.site",
+        studio_link="http://studio.my.edx.site",
+        # TODO: add in once implemented
+        customise_link="http://ocim.site/customise/",
+    )
+    html_email_helper(
+        template_base_name='emails/welcome_email',
+        context=context,
+        subject=settings.WELCOME_EMAIL_SUBJECT,
+        recipient_list=(email,)
+    )
```

Comment:
Is it something that should be removed (as well as the URLs) before merging?

Reply:
I was wondering if it's OK to keep it around as a way to test the email setup. I can remove it also.

Reply:
It's fine to keep this commented out here with a note that you can uncomment it for testing.
Or we can add this part of code to testing instructions in case we wanted to test it locally. I'll leave it up to you :)

Reply:
@Agrendalath commented out code quickly goes out of date since it isn't ever quality checked. 

I'll move it to the testing instructions isntead.